.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -3,22 +3,7 @@ set -exu
 
 ZENKO_NAME=${1:-end2end}
 COMMAND=${2:-"premerge"}
-# Get the current number of vCPUs
-VCPU_COUNT=$(nproc)
-# Calculate the number of parallel runs based on the vCPU count
-# - If there are only 1 vCPU set parallel runs to 1.
-# - If there are between 1 and 4 vCPUs, use 0.5 parallel runs per vCPU.
-# - If there are more than 4 vCPUs, use 0.5 parallel runs for the first 4 vCPUs and 1 parallel run for
-#   each additional vCPU, as Zenko is not CPU bound.
-PARALLEL_RUNS=$(awk -v vcpu=$VCPU_COUNT 'BEGIN {
-  if (vcpu <= 0) {
-    print 1
-  } else if (vcpu >= 1 && vcpu <= 4) {
-    print int(vcpu * 0.5)
-  } else {
-    print (4 * 0.5) + ((vcpu - 4) * 1)
-  }
-}')
+PARALLEL_RUNS=4
```

Comment:
why hard-code to 4, instead of using `nproc` as before?

Reply:
We have a static 4 in the upper branches, it's to have everything consistent. I saw little benefits of having more anyway, since we added more tests, it has a cpu impact that can increase the "flakies" rate

Reply:
i am fine with a max of `4` (or simplifying the computation to `max(1, ncore/2)`) : my concern is more that we may use 4 cores even when running on a smaller vm (possibly locally)

Reply:
I see, I updated the logic to keep the dynamic aspect, but simplify it: https://github.com/scality/Zenko/pull/2085/commits/64fce00face4d8d85ebbae0159e3d589d031c60f

No need to have the 0.5 vcpu per vcpu above 4, as the old computation was relevant when we used to run CTST in CLI mode. SDK is lighter.